register.js

```diff
@@ -0,0 +1,13 @@
+var fs = require('fs');
+var removeTypes = require('./index');
+
+var prev = require.extensions['.js'];
+
+require.extensions['.js'] = function(module, filename) {
+  if (filename.indexOf('node_modules') === -1) {
+    var source = fs.readFileSync(filename, 'utf8');
+    module._compile(removeTypes(source), filename);
+  } else {
+    prev(module, filename);
```

Comment:
Interesting, I tested it and it's fine with Node 5. Might be a different node version then...